src/test/java/lesson6/task8/IdValidatorTest.java

```diff
@@ -0,0 +1,19 @@
+package lesson6.task8;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class IdValidatorTest {
+
+    @Test
+    void shouldValidatePersonalId() {
```

Comment:
name of the method is shouldValidatePersonalId but you are expecting here also exception throw, it should be spiitted into separate tests